theano/sandbox/linalg/ops.py

```diff
@@ -280,8 +244,6 @@ def transinv_to_invtrans(node):
 @register_stabilize
 @local_optimizer([Dot, Dot22])
 def inv_as_solve(node):
-    if not imported_scipy:
```

Comment:
Not quite sure if allowing the execution to go through here is the right thing to do, since `scipy.linalg` is not imported.

But since the library is never used, I suppose the check is not needed anyway, and an alternative has been found.